File: front-end-user/src/app/models/answer-sheet.ts
import {Choice} from './choice';

export class AnswerSheet {
  questionId: number;
  choices: Choice[];
  point: number;

  constructor(id: number, choices: Choice[], point: number) {
    this.questionId = id;
    this.choices = choices;
    this.point = point;
  }
}

File: source code/backend/src/main/java/com/thanhtam/backend/dto/AnswerSheet.java
package com.thanhtam.backend.dto;

import com.thanhtam.backend.entity.Choice;
import com.thanhtam.backend.entity.Exam;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AnswerSheet {
    private Long questionId;
    private List<Choice> choices;
    private Integer point;
}
